src/menu-private.h

```diff
@@ -147,8 +148,9 @@ gboolean brisk_menu_window_setup_session(BriskMenuWindow *self);
 
 /* Settings */
 void brisk_menu_window_init_settings(BriskMenuWindow *self);
+void brisk_menu_applet_init_settings(BriskMenuWindow *self, BriskMenuApplet *applet);
 void brisk_menu_window_update_search(BriskMenuWindow *self, SearchPosition pos);
-void brisk_menu_window_pump_settings(BriskMenuWindow *self);
+void brisk_menu_window_pump_settings(BriskMenuWindow *self, BriskMenuApplet *applet);
```

Comment:
Makes sense - I haven't done C development in well over a decade, so I spent most of my time moving things around just to get it to compile without errors. I've since revisited this and realized I'm coupling the BriskMenuWindow with the BriskMenuApplet unnecessarily. I've since fixed this and effectively let the Applet handle the applet settings and leave the Window alone.